public/Lab-Files/Lesson04/Challenges/Team-Challenge-1/index.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+</head>
+<body>
+    <header>
+        <h1>Recent From Blog</h1>
+    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Facere, iusto, dignissimos veritatis quaerat aut aliquam.</p>
+    </header>
+    
+    <aside class="left">
+        <div>
+        <img class="one" src="img/keyboard.JPG" alt="pic of keyboard">
+        <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, repudiandae aspernatur a nulla modi quod.</p>
+        </div>
+    </aside>
+    
+    <aside class="right">
+        <div>
+            <img class="rightside" src="img/laptop.jpg" alt="pic of laptop">
+            <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, repudiandae aspernatur a nulla modi quod.</p>
+        </div>
+    
+    
+        <div>
+            <img class="rightside" src="img/desk.jpg" alt="pic of desk">
+            <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, repudiandae aspernatur a nulla modi quod.</p>
+        </div>
+    
+
+        <div>
+            <img class="rightside" src="img/window.jpg" alt="pic of window"/>
+            <h3>Lorem ipsum dolor sit amet, consectetur adipisicing.</h3>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Commodi, repudiandae aspernatur a nulla modi quod.</p>
+        </div>
+    </aside>
+    
+    <hr style="width:80%; height:5px;" />
```

Comment:
Try to keep all of your CSS in the style sheet. Line 44, you added an inline CSS for the hr.